### ISSUE TITLE
Dizzay crash when the video isn't available anymore

### ISSUE THREAD

**Issue by Sceat** [goto-bus-stop/dizzay#17]
# Dizzay crash when the video isn't available anymore
### Stack

![](http://i.imgur.com/Afc60l9.png)
### Stack (raw)

```
C:\Users\MrSceat\dizzay\lib\mplayer.js:67
      throw e;
      ^

Error: Command failed: youtube-dl --get-url -f bestaudio https://youtube.com/watch?v=rqdZsRGktPA
ERROR: rqdZsRGktPA: YouTube said: "FuntCase, Cookie Monsta,..."
The YouTube account associated with this video has been terminated due to multiple third-party notifications of copyright infringement.

    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:492:12)
```

---
### To reproduce
- Start dizzay
- Choose a track on your playlist that play an unavailable youtube video

**Comment by Sceat** [comment]
Another possible unavailable crash

### Stack

![](http://i.imgur.com/6VzWLWC.png)

### Stack (row)

```
λ node dizzay -m mplayer -r edtentertainment
C:\Users\MrSceat\dizzay\lib\mplayer.js:67
      throw e;
      ^

Error: Command failed: youtube-dl --get-url -f bestaudio https://youtube.com/watch?v=lWfJOdKfaeU
ERROR: lWfJOdKfaeU: YouTube said: This video does not exist.

    at ChildProcess.exithandler (child_process.js:206:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:852:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
```

**Comment by goto-bus-stop** [comment]
Should be fixed as of dizzay@0.8.0!
